usr/src/uts/i86pc/os/intr.c

```diff
@@ -471,6 +472,7 @@
 #include <sys/hypervisor.h>
 #endif
 
+boolean_t kernel_preempt_enable = B_FALSE;
```

Comment:
That's fair, the lack of support for other platforms and the off-by-default behavior are both tricky aspects of the `sys_rtt_kpreempt_enable` change. I'd be ok dropping that part of the change and just carrying it locally. I'll update the illumos bug with this and the now-on-by-default explicit `kpreempt()`. Thanks for the review!